recipe/meta.yaml

```diff
@@ -17,12 +17,13 @@ source:
     - patches/disable_jpeg12.patch
 
 build:
-  number: 3
+  number: 4
   skip: True  # [win and vc<14]
 
 requirements:
   build:
     - cmake  # [win]
+    - pkg-config  # [unix]
```

Comment:
I'm now getting `OpenJPEG support:          yes` on the CI's with this which is a good start...

Reply:
Not sure if I need to add `run_test.sh` to the test section here also to get it to run...